lib/flatpak-installation.c

```diff
@@ -2006,7 +2063,7 @@ flatpak_installation_fetch_remote_metadata_sync (FlatpakInstallation *self,
 /**
  * flatpak_installation_list_remote_refs_sync:
  * @self: a #FlatpakInstallation
- * @remote_name: the name of the remote
+ * @remote: the name or URI of the remote
```

Comment:
If this is what we want to do, the actual variable name should be updated as well. But this isn't the API used by `flatpak_builtin_ls_remote` so it's at least not the only place this change should be made. But this behavior is broken on master anyway.

Reply:
I reported that as https://github.com/flatpak/flatpak/issues/1588

Reply:
This is a documentation change, the functionality of listing by URI was already supported (possibly a misfeature). Take into account all these patches have been reviewed by PhilipW.

Reply:
The patch is wrong regardless of the fact that it's been reviewed. It's wrong because it changes the documentation to no longer match the actual variable name in the code. Anyway, I'm looking at fixing #1588 right now, then this will make a bit more sense.

Reply:
I pushed a fixup for this